client/Dockerfile.dev

```diff
@@ -0,0 +1,10 @@
+FROM node:10-alpine AS alpine
+WORKDIR /app
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
```

Comment:
The line below has to be modified. We're using Yarn and it has `yarn.lock` instead of `package-lock.json`.